p2p/server.go

```diff
@@ -631,6 +638,23 @@ running:
 			}
 		case pd := <-srv.delpeer:
 			// A peer disconnected.
+			// if sql db available, remotedRequested, and the discReason is TooManyPeers
+			// update the node meta info's too many peer counter
+			// TODO: split the too_many_peer_counter into 2 fields, one for p2p another for eth
+			/*
+				if srv.DB != nil && pd.requested {
```

Comment:
Why include this commented out code? Are you planning to add it in later?

Reply:
Yes. This is where `srv.addNodeMetaInfo` case 3 (mentioned in #23) occurs. I (probably) meant to add this in a different PR. I will just remove this for now.